libs/util/git/src/git-client.service.ts

```diff
@@ -224,11 +227,13 @@ export class GitClientService {
             repositoryName,
             branchName,
             commitMessage,
-            pullRequestBody,
             preparedFiles,
             baseBranch,
             repositoryGroupName,
             pullRequestTitle,
+            pullRequestBody:
+              getServicePullRequestMessage(resourceName, resourceUrl) +
+              pullRequestBody,
```

Comment:
@Meyazhagan sorry, I was probably too vague. I meant that the change should not live in the `util-git` library at all. 
My expectation is to see only changes in the services.